addons/nametags/XEH_postInit.sqf

```diff
@@ -23,5 +23,24 @@ GVAR(ShowNamesTime) = -10;
 [29, [false, false, false]], false] call cba_fnc_addKeybind; //LeftControl Key
 
 
-// Draw handle
-addMissionEventHandler ["Draw3D", {_this call FUNC(onDraw3d);}];
+// Draw handle on start if set
+if (GVAR(showPlayerNames) > 0) then {
+    GVAR(drawHandler) = addMissionEventHandler ["Draw3D", {_this call FUNC(onDraw3d);}];
+};
+
+// Set the EH which waits for a setting to be changed, so that the effect is shown immediately
+if (!GVAR(showPlayerNamesForce)) then {
```

Comment:
@esteldunedain is working on the init thingy, was decided to leave as is for now.